### PR TITLE
Stylelint 10.x upgrade

### PR DESCRIPTION
**Pull request by pascalduez**
Stylelint 10.x [changelog](https://github.com/stylelint/stylelint/blob/master/CHANGELOG.md#1000)

To be honest I  can't make any sense of those tests `columns`:  
For instance in `.foo { text-align: justify; }`   
`3` would refer to the first `o`, but even the previous `4` would refer to the last `o`.  

Also this probably means a new major version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "a11y"
   ],
   "engines": {
-    "node": ">=6"
+    "node": ">=8.7.0"
```
Comment (pascalduez) [inline package.json]: https://github.com/stylelint/stylelint/blob/b35c55f6144d0cb03924498f275cc39c39de3816/package.json#L38-L40

Comment (YozhikM) [reply package.json]: I agree, we must comply with the main project (Stylelint) in there.

**Comment by YozhikM** [comment]
Great work, @pascalduez ! Thanks for this PR.

**Comment by YozhikM** [comment]
Tests passed. I think we can merge this and publish a new version after I check the library on the CSS test base.

File: src/rules/line-height-is-vertical-rhythmed/__tests__/index.js [hunk 2]
```diff
@@ -33,43 +33,43 @@ testRule(rule, {
...
     },
     {
       code: 'p { font-size: 23px; line-height: 23px; }',
       message: messages.expected('p'),
       line: 1,
-      column: 4,
+      column: 3,
     },
     {
-      code: 'a { font-size: 24px; line-height: 1; }',
```
Comment (pascalduez) [inline src/rules/line-height-is-vertical-rhythmed/__tests__/index.js]: @YozhikM Youps sorry, that's an error.

File: src/rules/line-height-is-vertical-rhythmed/__tests__/index.js [hunk 3]
```diff
@@ -33,43 +33,43 @@ testRule(rule, {
...
+      code: 'a { font-size: 23px; line-height: 1; }',
```
Comment (pascalduez) [inline src/rules/line-height-is-vertical-rhythmed/__tests__/index.js]: @YozhikM Youps sorry, that's an error.

**Pull request merged**